[msx] testStarted testRecoveryInProgressException
2019-12-19 16:33:50,080 [main] INFO  datanode.DataNode (DataNode.java:<init>(464)) - [msx-restart] DataNode new conf
2019-12-19 16:33:50,289 [main] INFO  datanode.DataNode (DataNode.java:shutdown(1995)) - [msx-restart] DataNode stop
[msx] testFinished testRecoveryInProgressException testresult=1
